### ISSUE TITLE
Azure Powershell Modules fail to load with custom deployment scripts

### ISSUE THREAD

**Issue by shaunmarx** [OctopusDeploy/Issues#4637]
There appears to be an issue with how Powershell modules are loaded when using custom deployment scripts for some customers. Affects versions 2018.5.x and later (even after all module loading fixes)

**Error:**
```
Get-AzureRmEnvironment : The ‘Get-AzureRmEnvironment’ command was found in the module ‘AzureRM.Profile’, 
but the module could not be loaded. For more information, run ‘Import-Module AzureRM.Profile’.
```

Unfortunately we have been unable to reproduce and determine the exact cause of the issue. The error message seems to indicate incorrect versions of modules being loaded, however even after close inspection modules paths and loaded modules all seem to be correct without any rational explanation for things failing specifically for custom deployment scripts.

At this point in time no workaround seems to exist as setting the global module flag or even replacing the modules in-line do not appear to fix the issue.

Sources:
https://help.octopus.com/t/web-deploy-gives-a-powershell-error-after-upgrade-to-latest-version/20128/28
https://help.octopus.com/t/issue-with-azurerm-profile-module/20379

**Comment by shaunmarx** [comment]
Closing as no more cases reported and it would appear that the pipeline fixes as of 2018.5 has addressed this particular problem. This in combination with related issues such as OctopusDeploy/Issues#4647 made it appear that no workaround existed, which is not the case.

**Comment by OliG19** [comment]
We are still seeing this as an issue even on version 2018.6.5, is it possible for this to be re-opened?
